static/monorail.yml

```diff
@@ -1432,7 +1437,7 @@ paths:
         - name: q
           in: query
           description: query object to pass through to waterline.
-          required: false
+          required: true
```

Comment:
@zyoung51 comment ^^ still holds true when required is set to true.  The python functional tests run tests without the 'q' parameter, both library and functional tests will need to be updated if we force 'q' parameter.